packages/lesswrong/components/notifications/NotificationsList.jsx

```diff
@@ -9,7 +9,7 @@ class NotificationsList extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      lastNotificationsCheck: props.currentUser.lastNotificationsCheck,
+      lastNotificationsCheck: (props.currentUser.lastNotificationsCheck || ""),
```

Comment:
If we have this line, we should add currentUser as a required prop in proptypes, otherwise the check should be `props.currentUser && props.currentUser.lastNotificationsCheck || ""`